python/ray/train/examples/mosaic_cifar10_example.py

```diff
@@ -82,7 +81,7 @@ def train_mosaic_cifar10(num_workers=2, use_gpu=False):
     from ray.train.mosaic import MosaicTrainer
 
     trainer_init_config = {
-        "max_duration": "2ep",
+        "max_duration": "5ep",
```

Comment:
Nit: `max_duration` should be a `train_mosaic_cifar10` parameter. 

Readers won't be able to understand why
```
 assert result["epoch"][result.index[-1]] == 4
```
unless they look at the implementation for `train_mosaic_cifar10`.